app/components/LongCode.tsx

```diff
@@ -50,11 +50,22 @@ const LongCode: FC<LongCodeProps> = ({ code, codeClass }: LongCodeProps) => {
   return (
     <Box data-testid="long-code-code" className={codeClass} aria-hidden="true">
       {codeLines.map((line, i) => {
-        return (
-          <Box component="span" key={[line, i].join('|')}>
+        return (i === codeLines.length - 1) ? (
+          <Box
+            component="span"
+            key={[line, i].join('|')}
+            style={{
+              alignSelf: 'flex-end',
+            }}
+          >
             {line}
           </Box>
-        );
+        )
+          : (
+            <Box component="span" key={[line, i].join('|')}>
+              {line}
+            </Box>
+          );
```

Comment:
I'm fine with the inline style. A cleaner way to do this may be: 

```
return (
  <Box
    component="span"
    key={[line, i].join('|')}
    style={i === codeLines.length - 1 ? { alignSelf: 'flex-end' }: null}
  >
    {line}
  </Box>
);
```

or to keep it as a class

```
return (
  <Box
    component="span"
    key={[line, i].join('|')}
    className={i === codeLines.length - 1 ? 'flexSoHard': ''}
  >
    {line}
  </Box>
);
```

if things really get complicated. You could introduce `clsx`. 

```
return (
  <Box
    component="span"
    key={[line, i].join('|')}
    className={clsx({ flexSoHard: i === codeLines.length - 1) }
  >
    {line}
  </Box>
);
```